assessments-service/src/main/java/com/transformuk/hee/tis/assessment/service/service/impl/AssessmentServiceImpl.java

```diff
@@ -257,49 +257,30 @@ public AssessmentDTO findOne(Long id) {
 
   @Override
   @Transactional
-  public boolean deleteTraineeAssessment(Long assessmentId, Long traineeId) {
-    Preconditions.checkNotNull(traineeId);
+  public boolean deleteAssessment(Long assessmentId) {
```

Comment:
Seems there're no changes for `deleteAssessment` and `deleteTraineeAssessment`. But on this page they look messy to me.

Reply:
the deleteAssessment and deleteTraineeAssessment have both remained the same they appear to have been shuffled during the process, the front end has been routed from softDeleteTraineeAssessment which has been deleted to deleteTraineeAssessment and deleteAssessment is used by another service